README.md

```diff
@@ -228,121 +232,52 @@ and proxy the connection to the insecure backend.
 Advanced Features
 =================
 
-### Metrics & profiling
+### Access Control Flags
 
-Ghostunnel has a notion of "status port", a TCP port (or UNIX socket) that can
-be used to expose status and metrics information over HTTPS. The status port
-feature can be controlled via the `--status` flag. Profiling endpoints on the
-status port can be enabled with `--enable-pprof`.
+Ghostunnel supports different types of access control flags in both client and
+server modes.  All checks are made against the certificate of the client or
+server. Multiple flags are treated as a logical disjunction (OR), meaning
+clients can connect as long as any of the flags matches. Ghostunnel is
+compatible with [SPIFFE][spiffe] [X.509 SVIDs][svid].
 
-The X.509 certificate on the status port will be the same as the certificate
-used for proxying (either the client or server certificate). This means you can
-use the status port to inspect/verify the certificate that is being used, which
-can be useful for orchestration systems.
+See [ACCESS-FLAGS](docs/ACCESS-FLAGS.md) for details.
 
-Example invocation with status port enabled:
+[spiffe]: https://spiffe.io/
+[svid]: https://github.com/spiffe/spiffe/blob/master/standards/X509-SVID.md
 
-    ghostunnel client \
-        --listen localhost:8080 \
-        --target localhost:8443 \
-        --keystore test-keys/client-keystore.p12 \
-        --cacert test-keys/cacert.pem \
-        --status localhost:6060
+### Certificate Hotswapping
 
-Note that we set the status port to "localhost:6060". Ghostunnel will start an
-internal HTTPS server and listen for connections on the given host/port. You
-can also specify a UNIX socket instead of a TCP port.
+To trigger a reload, simply send `SIGUSR1` to the process or set a time-based
+reloading interval with the `--timed-reload` flag. This will cause ghostunnel
+to reload the certificate and private key from the files on disk. Once
+successful, the reloaded certificate will be used for new connections going
+forward.
 
-How to check status and read connection metrics:
+Additionally, ghostunnel uses `SO_REUSEPORT` to bind the listening socket on
+platforms where it is supported (Linux, Apple macOS, FreeBSD, NetBSD, OpenBSD
+and DragonflyBSD). This means a new ghostunnel can be started on the same
+host/port before the old one is terminated, to minimize dropped connections (or
+avoid them entirely depending on how the OS implements the `SO_REUSEPORT`
+feature).
 
-    # Status information (produces JSON output)
-    curl --cacert test-keys/cacert.pem https://localhost:6060/_status
+Note that if you are using an HSM/PKCS#11 module, only the certificate will
```

Comment:
should we embolden this caveat?

Reply:
So I assume ghostunnel supports new certs that are issued using the same private key?

Reply:
Yes. I can try to make this clearer maybe? It reloads the cert, so it can be a new/updated cert, but must be based on the same public/private key pair.

Reply:
yeah, I think that would make it clearer.

Reply:
Clarified the language here